nuxeo-dm/nuxeo-platform-webapp/src/main/resources/web/nuxeo.war/incl/nuxeo_javascript_context.xhtml

```diff
@@ -0,0 +1,13 @@
+<ui:fragment xmlns="http://www.w3.org/1999/xhtml"
+  xmlns:h="http://java.sun.com/jsf/html"
+  xmlns:ui="http://java.sun.com/jsf/facelets">
+
+  <h:outputScript>
```

Comment:
Not something to be done right now but since this could be very useful for the undergoing polymer work we should review it and at least place it in a `nuxeo` namespace/object at some point.